scripts/run_integration_tests.sh

```diff
@@ -119,13 +152,16 @@ set +e # Don't exit this script if there is a diff or the logs endpoint fails
 echo "Fetching logs for invocations and comparing to snapshots"
 for handler_name in "${LAMBDA_HANDLERS[@]}"; do
     for runtime in "${RUNTIMES[@]}"; do
-        function_name="${handler_name}_${runtime}"
-        function_snapshot_path="./snapshots/logs/${function_name}.log"
-
+        function_name="${handler_name}_node"
+        function_snapshot_path="./snapshots/logs/${handler_name}_${runtime}.log"
+        serverless_runtime=$runtime[0]
+        nodejs_version=$runtime[1]
+        run_id=$runtime[2]
         # Fetch logs with serverless cli, retrying to avoid AWS account-wide rate limit error
         retry_counter=0
         while [ $retry_counter -lt 10 ]; do
-            raw_logs=$(serverless logs --stage $run_id -f $function_name --startTime $script_utc_start_time)
+            raw_logs=$(NODE_VERSION=${!nodejs_version} RUNTIME=$runtime SERVERLESS_RUNTIME=${!serverless_runtime} \
```

Comment:
Why do we need to know the node version, runtime, etc. to fetch the logs?

Reply:
we don't :/ but my understanding is that serverless.yaml is fully evaluated at each serverless cmd invocation.
And since this file is now parametrized with some envs, I have noticed that we have to pass them otherwise we end with ```<   A valid environment variable to satisfy the declaration 'env:NODE_VERSION' could not be found.```
```<   A valid environment variable to satisfy the declaration 'env:SERVERLESS_RUNTIME' could not be found.```

Reply:
That makes sense!